ot/plot.py

```diff
@@ -14,14 +14,16 @@
 
 import numpy as np
 import matplotlib.pylab as pl
-from matplotlib import gridspec
 
 
-def plot1D_mat(a, b, M, title=''):
-    r""" Plot matrix :math:`\mathbf{M}`  with the source and target 1D distribution
+def plot1D_mat(a, b, M, title='', a_label='Source distribution',
```

Comment:
thanks for the big and necessary update but keep in mind that yteh function must keep a very similar behavior by default than teh previous version.

That is the color and source/target should be  the same as previous version . It is OK to have parameter that change this default behavior and you can use the values you want in your examples but this update should not change too much the other examples that use the function (except for the fill of the densities it is much better than the previous plot).

Reply:
I made the default behaviour basically identical to the old ones (just quality upgrades, everything is in the same place), and the 'xy' version can be used with `plot_style` which defaults to yx. Colours also default to blue and red :)